KOGITO-7312: Patch PR for KOGITO-7311 with doc improvements in Mocking HTTP CloudEvents sink with Wiremock guide

hmanwani-rh opened the pull request:
<!-- Please don't forget your JIRA link -->
**JIRA:**https://issues.redhat.com/browse/KOGITO-7312

<!-- If you don't have a JIRA link, please provide a short description of what this PR does -->
**Description:**Patch PR for KOGITO-7311 with doc improvements in Mocking HTTP CloudEvents sink with Wiremock guide

<!-- Link to related PRs: -->
https://github.com/kiegroup/kogito-docs/pull/55/

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category

hmanwani-rh commented:
@nmirasch Please review the changes in the PR and let me know if the changes look good.

nmirasch commented on serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:
```diff
@@ -264,29 +266,33 @@ Now, the test needs to check the sink is receiving the expected events with:
 ----
 
-After assert the validations on received events, the test ends with success and the used WireMockServer will stop.
+After declaring the verfications on received events, the test successfully ends and the WireMockServer stops. 
```
After declaring the verification ..

hmanwani-rh replied:
Done

nmirasch commented on serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:
```diff
@@ -264,29 +266,33 @@ Now, the test needs to check the sink is receiving the expected events with:
-Checking if the sink (WireMockServer) receives two events for that order id with:
+The following example shows how to check if the sink (`WireMockServer`) receives two events for the same order ID:
+
+.Example if sink recevies the events
```
Example if sink receives..

hmanwani-rh replied:
Done

nmirasch commented on serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:
```diff
@@ -151,84 +153,83 @@ public class VerifyWorkflowExecutionIT { <2>
 
-Fraud Handling will produce a new `FraudEvaluation` event if the order is above 1000 USD.
-Any other system or service in the architecture can then read this event and react upon it, like canceling the order for example.
+The Fraud Handling worklflow produces a `FraudEvaluation` event if the received order is more than 1000 USD. In the workflow architecture, any other system or service can read the `FraudEvaluation` event and react upon it, such as canceling the order. 
```
The Fraud Handling workflow..

hmanwani-rh replied:
Done

nmirasch commented on serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:
```diff
@@ -28,71 +27,74 @@ Mocking HTTP CloudEvents Sink with WireMock
-The purpose of this guide is to mock that service to verify if the Cloud Events are correctly received by the sink.
-The {wiremock_url}[*WireMok*] framework will add the mocked server that will verify the different Cloud Events received by the sink in the Serverless Workflow service execution.
---
+When you create an event source, you can specify a sink where events are sent to, from the source. A sink is an transferable or a callable resource that can receive incoming events from other resources. The examples of sink include Kubernetes deployments, Knative services, channels, and brokers.
```
... A sink is a transferable..

hmanwani-rh replied:
Done

nmirasch reviewed (approved):
@hmanwani-rh I just added small typos but the content looks good to me. Thank you very much, great job!

hmanwani-rh commented:
> @hmanwani-rh I just added small typos but the content looks good to me. Thank you very much, great job!

@nmirasch Thanks for reviewing the changes and catching typos. I've fixed all the typos. 
Therefore, merging this PR.
